aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -2716,6 +2716,54 @@ void dequantize_tensor_per_channel_float_qparams_cpu(
       });
 }
 
+void quantize_tensor_per_tensor_affine_sub_byte_cpu(
```

Comment:
Should this function take the bit_rate (lets rename it to bitwidth) as input so that we can reuse it for 2 bits later?

Reply:
I think we can use the dispatcher for this, we can add a bitwidth field in the dispatch so this function can be generalized.